docs/oauth2-oidc/device-authorization.mdx

```diff
@@ -0,0 +1,132 @@
+---
+id: device-authorization
+title: Device Authorization
+sidebar_label: Device authorization flow
+---
+
+The OAuth 2.0 Device Authorization Grant (also known as Device Flow) is OAuth 2.0 extension that enables devices with no browser
+or limited input capability to obtain an access token. It enables users to authenticate devices with limited input capabilities,
+such as smart TVs, gaming consoles, or IoT devices, by delegating the authentication process to another device with a full browser
+(e.g., a smartphone or computer).
+
+This document provides an overview of the Device Authorization Grant, a step-by-step example of its implementation, configuration
+options, and guidance on creating custom user interfaces for the verification screen.
+
+## Overview of the Flow
+
+Here is the high-level overview for the Device Authorization Flow:
+
+1. The device requests to be authorized from the Authorization server.
+2. The user is instructed to visit a URL on a different device and is given a user code.
+3. On a different device the user, visits the URL, provides the user code, logs in and grants access to the device.
+4. The device polls the Authorization server. Once the user authenticates and grants access, an access token is returned that can
+   be used to access the protected resource.
+
+### Step 1: Device Requests Authorization
```

Comment:
generally not a fan of "Step X:" in headlines. 
However I can't think of a better way to express this here 🤔

Reply:
I used the auth code docs as inspiration. Perhaps I can just remove the `Step X` prefix? I will give it a shot